components/security/org.wso2.carbon.security.mgt/src/main/java/org/wso2/carbon/security/keystore/KeyStoreAdmin.java

```diff
@@ -324,7 +330,7 @@ public void importCertToStore(String fileName, String certData, String keyStoreN
                         .generateCertificate(new ByteArrayInputStream(bytes));
             } catch (CertificateException e) {
                 log.error(e.getMessage(), e);
-                throw new SecurityConfigException("Invalid format of the provided certificate file");
+                throw new SecurityConfigException("Invalid format of the provided certificate file", e);
```

Comment:
since we log exception don't need to add , e) to the exception. Just throw exception with the message.